indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -386,7 +386,22 @@ def can_access(self, user):
             return True
         if any(x.person.user == user for x in self.person_links):
             return True
-        return False
+        return self.can_review(user)
+
+    def can_review(self, user):
```

Comment:
let's add it when we need it? in this case we don't really care about the track and it'd be easy to extend this method once we need it somewhere